node/hyperbahn/handler.js

```diff
@@ -24,10 +24,14 @@
 
 var assert = require('assert');
 var util = require('util');
+var fs = require('fs');
+var path = require('path');
 var setTimeout = require('timers').setTimeout;
 
 var Errors = require('../errors.js');
 var TChannelJSON = require('../as/json');
+var TChannelThrift = require('../as/thrift');
```

Comment:
Let's not use thrift until `thriftify` is gone. I want to use the new `thriftrw` only.